cypress/integration/003-usage.js

```diff
@@ -25,20 +24,18 @@ describe('usage page ', function () {
     cy.get('#main-content form button.naturescot-forward-button').click();
     cy.url().should('include', '/trap-registration-number');
   });
-
   it('no button should navigate to trap no-usage page', function () {
     cy.visit('/usage');
     cy.get('#main-content form input[type="radio"][value="no"]').click();
     cy.get('#main-content form button.naturescot-forward-button').click();
     cy.url().should('include', '/no-usage');
   });
-
```

Comment:
...or these ones. I'm pretty sure they were there to visually distinguish separate tests. Did this get flagged by the linter?

Reply:
Resolved in 911481548a8e2d58a41903741679c291d257b844.